src/main/java/it/arcade/hospital/controller/TestController.java

```diff
@@ -0,0 +1,11 @@
+package it.arcade.hospital.controller;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+public class TestController {
+    @GetMapping("/test")
```

Comment:
Task requirements is to unlock endpoint in security. At this moment it require authentication